packages/typescript/src/components/ObjectExpression.tsx

```diff
@@ -110,11 +108,15 @@ export function ObjectProperty(props: ObjectPropertyProps) {
 
   let sym = undefined;
   if (props.refkey && props.name) {
-    sym = createTSSymbol({
-      name: props.name,
-      refkey: props.refkey,
+    sym = new TSOutputSymbol(props.name, {
+      refkeys: props.refkey,
       flags: OutputSymbolFlags.StaticMember,
     });
+
+    moveTakenMembersTo(sym);
+  } else {
+    // noop
```

Comment:
Why is takeSymbols() here a noop?

Reply:
Since we are presently conditionally making the symbol when we have a refkey and a name, we don't need to do any work when we take a symbol, but we still need to take them otherwise the symbols might end up in a parent somewhere would be incorrect.